modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java

```diff
@@ -320,7 +320,9 @@ private ObjectField _toObjectField(
 				HashMapBuilder.put(
 					"delete",
 					() -> {
-						if (!objectField.isDeletionAllowed()) {
+						if (!objectField.isDeletionAllowed() ||
+							objectField.isSystem()) {
```

Comment:
We don't need to know if the object field is system in [this case](https://github.com/liferay-objects/liferay-portal/pull/2513/files#diff-a50d68735e9840b6a835fed264e1b77700dcf952afe1eb61018ab8462a691a16R992-R994). That's why the `objectField.isSystem()` is outside the method.
Do you believe that we need to check if the object field is system in this case?

I believe that we don't need to check it because in the next story we will allow to create system object relationships, so this logic will be also used for system object fields